### PR TITLE
Automatic conversion added for diverse currencies closes #2

### LINKED ISSUE #2: Implement Currency conversion
It would be very beneficial if the currency in USD may be converted to other currencies, like INR

### PR DESCRIPTION
**Pull request by seraco**
I have done the conversion from USD to different currencies by using http://fixer.io/

Closes #2

### CODE REVIEW

**Comment by recrsn** [comment]
Thanks @seraco 

Just 1 little job before I merge this, can you support showing the currency symbols before the value? Also, it is okay to make the list of currencies static for now. We can update the code, when fixer.io adds support for more currencies.

And yes, please fix those conflicts too!

Also if you could edit your commit/PR messages to auto-close #2 , details given here: https://help.github.com/articles/closing-issues-using-keywords/

File: app/scripts/main.js [hunk 1]
```diff
@@ -86,5 +95,10 @@ $(function () {
     })
   };
   //load ec2 pricing data
-  $.getScript('https://spot-price.s3.amazonaws.com/spot.js');
+  $.get( 'http://api.fixer.io/latest?base=USD', function( data ) {
```
Comment (recrsn) [inline app/scripts/main.js]: Use "https" for obvious reasons.

**Review by recrsn — CHANGES_REQUESTED** [review]

**Review by recrsn — APPROVED** [review]

**Comment by seraco** [comment]
Closes #2

**Comment by seraco** [comment]
I am not able to automatically close the issue as the documentation explains.

**Comment by recrsn** [comment]
>I am not able to automatically close the issue as the documentation explains.

No problem. I edited your PR message to set to auto-close #2. You just need to include Closes/Fixes #2 somewhere in PR/commit messages. And when the PR is merged, the issue is automatically closed.

**Pull request merged**